app/src/main/AndroidManifest.xml

```diff
@@ -565,6 +565,11 @@
             </intent-filter>
         </receiver>
 
+        <receiver
+            android:name="androidx.mediarouter.media.MediaTransferReceiver"
+            android:exported="true">
+        </receiver>
```

Comment:
Adding this "enables media transfer among devices with system UI" which means that on the casting notification there is the option to switch where you are casting to.

https://developers.google.com/cast/docs/android_sender/output_switcher#enable_output_switcher_in_androidmanifestxml

| Before | After |
| --- | --- |
| ![Screenshot_20241025_134511](https://github.com/user-attachments/assets/5e2cf58e-3b46-4ddb-bb54-2b9b97f64b14) | ![Screenshot_20241025_134548](https://github.com/user-attachments/assets/6168a73f-addf-4131-97d9-6f91b3ef8887) |